resource/user/user.go

```diff
@@ -217,19 +183,10 @@ func (u *User) Check(context.Context, resource.Renderer) (resource.TaskStatus, e
 
 // Apply changes for user
 func (u *User) Apply(context.Context) (resource.TaskStatus, error) {
```

Comment:
I don't believe we can since we no longer keep `status` as a field of the task struct.